test/scenario_test/rtc_test.py

```diff
@@ -208,6 +208,28 @@ def test_10_rr_setup(self):
         self.__class__.g4 = g4
         self.__class__.g5 = g5
 
+        # add import/export policy to allow peers exchange routes within specific RTs
+        # later tests should not break due to RTC Updates being filtered-out
+        self.g3.local("gobgp policy neighbor add rr-clients-acme {} {}".format(
+            self.g3.peer_name(self.g4),
+            self.g3.peer_name(self.g5)))
+        self.g3.local("gobgp policy ext-community add rts-acme rt:^100:.*$ rt:^200:200$ rt:^300:300$")
+
+        self.g3.local('gobgp policy statement add allow-acme')
+        self.g3.local('gobgp policy statement allow-acme add condition neighbor rr-clients-acme')
+        self.g3.local('gobgp policy statement allow-acme add condition ext-community rts-acme')
+        self.g3.local('gobgp policy statement allow-acme add action accept')
+        self.g3.local('gobgp policy add allow-acme allow-acme')
+
+        self.g3.local('gobgp policy statement add allow-rtc')
+        self.g3.local('gobgp policy statement allow-rtc add condition afi-safi-in rtc')
+        self.g3.local('gobgp policy statement allow-rtc add action accept')
+        self.g3.local('gobgp policy add allow-rtc allow-rtc')
+
+        self.g3.local('gobgp global policy import add allow-rtc allow-acme default reject')
+        self.g3.local('gobgp global policy export add allow-rtc allow-acme default reject')
```

Comment:
I think this boils down, to what we've discussed previously. We'd like to contribute some tests for combined use-case, which don't test only single feature or small subset of them, but enable vpn, rr, rtc, add-path, graceful-restart, plus some policies, all at the same time. This would make merging upstream GoBGP easier. What do you think @iwaseyusuke , @fujita , @kishiguro ?

Reply:
Thanks for your reply.
I agree with having "some tests for combined use-case", but if we combines multiple features into a single `def test_xxx_setup(self)` method, all of the following test cases will be affected by these policies, then we can only test the use-case when these policies configured. I think most users will not always configure these policies and I'm afraid that we can not do test the situations which most users will do so.
So if we need to test these policy settings, I think we should have separated test cases (`test_xxx` methods) for these policies. [This patch](https://github.com/osrg/gobgp/commit/a7ecbaac46608455338367ea91f55c307fb589ed#diff-4af9c5f928d59cf389b2ad852a7c15deR70) introduced a "combined" test case as the separated method and restores the neighbor settings in order NOT to take effect for the following cases.

Reply:
I see your point. I'll remove this code from rtc test and make separate one that tests combination of rtc and policy with default reject (ensure afi-safi match).